src/apiClient.js

```diff
@@ -15,24 +13,8 @@ export default function APIClient(mpInstance, kitBlocker) {
             this.uploader = new BatchUploader(mpInstance, millis);
         }
         this.uploader.queueEvent(event);
-    };
-
-    this.shouldEnableBatching = function() {
-        // Returns a string of a number that must be parsed
-        // Invalid strings will be parsed to NaN which is falsey
-        var eventsV3Percentage = parseInt(
-            mpInstance._Helpers.getFeatureFlag(Constants.FeatureFlags.EventsV3),
```

Comment:
Are we going to remove `EventsV3` from `constants.ts` and other locations as well, since there's no need to choose v3 and v2?

Reply:
it's a good call out.  I think I'll remove the block about 8/12/2022 as well since it references it